nbs/api/00_core.ipynb

```diff
@@ -3109,10 +3109,29 @@
     "    add_toast(sess, tm, \"success\")\n",
     "    return P(\"I hate toast!\")\n",
     "\n",
-    "r = cli.get('/')\n",
-    "assert tm in r.text\n",
     "r = cli.get('/no-toast')\n",
-    "assert not tm in r.text"
+    "assert not tm in r.text\n",
+    "\n",
+    "r = cli.get('/')\n",
+    "assert tm in r.text"
```

Comment:
We flip the order of the two tests so that we can re-use the `r` variable in the next cell.